languages/PLI.groovy

```diff
@@ -23,15 +23,27 @@ buildUtils.createLanguageDatasets(langQualifier)
 // sort the build list based on build file rank if provided
 List<String> sortedList = buildUtils.sortBuildList(argMap.buildList, 'pli_fileBuildRank')
 
+if (buildListContainsTests(sortedList)) {
+	langQualifier = "pli_test"
+	buildUtils.createLanguageDatasets(langQualifier)
+}
+
 // iterate through build list
 sortedList.each { buildFile ->
 	println "*** Building file $buildFile"
 
+	// Check if this a testcase
+	isZUnitTestCase = (props.getFileProperty('pli_testcase', buildFile).equals('true')) ? true : false
```

Comment:
Hi @flpet-ibm,

I am missing the definition of `pli_testcase` in the `sample/application-conf/file.properties` file in your PR. Please add that like the `cobol_testcase` and also update the README for it.